src/tools/race/CMakeLists.txt

```diff
@@ -1,14 +1,20 @@
 find_package(Threads)
+find_package(LibRt)
 
 if (APPLE)
 	remove_tool(race "race tool does not compile under MAC OS X")
 else()
+	if (NOT LIBRT_FOUND)
+		remove_tool(race "race tool requires librt support")
```

Comment:
Thank you for checking! Can you update the PR? Please also document why `race` is excluded for APPLE.

Tests that use the race tool are in `tests/shell/check_race.sh` (currently disabled by default). Its easier to run the tool manually though.